packages/cli/lib/lib/webpack/webpack-base-config.js

```diff
@@ -63,7 +63,7 @@ function getSassConfiguration(...includePaths) {
  * @returns {import('webpack').Configuration}
  */
 module.exports = function createBaseConfig(env) {
-	const { cwd, isProd, isWatch, src, source } = env;
+	const { cwd, isProd, src, source } = env;
```

Comment:
Just a consistency thing. Easier to scan if we're not switching between `isProd` and `isWatch`.